src/components/ImageGeneration/GeneratedImageActions.tsx

```diff
@@ -97,6 +101,7 @@ export function GeneratedImageActions({
             <Button
               color="blue"
               size="sm"
+              px={isMobile ? 8 : 18}
```

Comment:
again, try to avoid using `isMobile`. Try to use tailwind classes. If tailwind doesn't have an exact match, see if the closest match will work. We are going to try to stick to tailwind sizes as much as possible for consistency. If you really need a different size, tailwind provides a way to create one-off classes inline

Reply:
```suggestion
              className="px-2 sm:px-4"
```